content/blog/2023-03-05-newsletter.md

```diff
@@ -0,0 +1,181 @@
+---
+title: The XMPP Newsletter February 2023
+date: 2023-03-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter, great to have you here again! This issue covers the month of February 2023. 
+Many thanks to all our readers and all contributors!
+
+Like this newsletter, many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, please consider saying thanks or help these projects! Interested in supporting the Newsletter team? Read more at the bottom.
+
+## XSF Announcements
+
+If you are interested in becoming an XSF member, the [listings are open until 19th February 2023](https://wiki.xmpp.org/web/Membership_Applications_Q1_2023).
+
+### XSF fiscal hosting projects
+
+The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+### Google Summer of Code 2023
+
+{{< figure src="/images/logos/gsoc-2023-logo.svg" caption="XSF and Google Summer of Code 2023" >}}
+
+We are considering to participate the Google Summer of Code 2023! If you are interested start [reading here](https://wiki.xmpp.org/web/Google_Summer_of_Code_2023).
+
+## XMPP Events
+
+- [XMPP Summit 25 and at FOSDEM 2023](https://xmpp.org/2023/01/xmpp-at-fosdem-2023/)
+- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+
+## XMPP Talks
+
+- [XMPP - End of the year 2022 talk](https://bbb.sfconservancy.org/b/ste-fdo-qen)
+- [Interoperable Chat, Dutch Healthcare and the Digital Services Act](https://fosdem.org/2023/schedule/event/interoperable_chat/) Winfried shares the many challenges faced while working on a standard for interoperable chat in healthcare that’s compatible with the EU’s new regulations.
+- [P10K: getting 10000 participants into a Jitsi meeting](https://fosdem.org/2023/schedule/event/jitsi_p10k/), Saúl from the Jitsi team reveals their interesting journey to reach 10000 participants in an XMPP-powered Jitsi video conference.
+- [Modernizing Authentication and Authorization in XMPP](https://fosdem.org/2023/schedule/event/modern_xmpp_auth/): Matthew Wild, founder of the Prosody and Snikket projects, explains why it’s time for us to look beyond passwords for authentication.
+
+## Articles
+
```

Comment:
nicfab wrote the article [Digital Service Act: EU Commission Questions and Answers on identification and counting of active recipients of the service under the DSA](https://notes.nicfab.eu/en/posts/dsa-faq/).